no-undef and  no-underscore-dangle uses single quotes

It's minor, but it would be nice to stick with one quotation style. Since most rules uses double quotes, i think `no-undef` and `no-underscore-dangle` should be changed.

```
   1:0   error  Use the global form of "use strict"           strict
  20:12  error  "args" used outside of binding context        block-scoped-var
  20:12  error  'args' is not defined                         no-undef
  26:1   error  Unnecessary semicolon                         no-extra-semi
  38:18  error  Unexpected dangling '_' in '__container__'    no-underscore-dangle
```

--- a/lib/rules/no-undef.js
+++ b/lib/rules/no-undef.js
@@ -59,8 +59,8 @@ function hasTypeOfOperator(node) {
 
 module.exports = function(context) {
 
-    var NOT_DEFINED_MESSAGE = "'{{name}}' is not defined.",
-        READ_ONLY_MESSAGE = "'{{name}}' is read only.";
+    var NOT_DEFINED_MESSAGE = "\"{{name}}\" is not defined.",
+        READ_ONLY_MESSAGE = "\"{{name}}\" is read only.";
 
     // TODO: Remove once escope is updated
     var hackedImports = [];

--- a/lib/rules/no-underscore-dangle.js
+++ b/lib/rules/no-underscore-dangle.js
@@ -35,7 +35,7 @@ module.exports = function(context) {
             var identifier = node.id.name;
 
             if (typeof identifier !== "undefined" && hasTrailingUnderscore(identifier)) {
-                context.report(node, "Unexpected dangling '_' in '" + identifier + "'.");
+                context.report(node, "Unexpected dangling \"_\" in \"" + identifier + "\".");
             }
         }
     }
@@ -45,7 +45,7 @@ module.exports = function(context) {
 
         if (typeof identifier !== "undefined" && hasTrailingUnderscore(identifier) &&
             !isSpecialCaseIdentifierInVariableExpression(identifier)) {
-            context.report(node, "Unexpected dangling '_' in '" + identifier + "'.");
+            context.report(node, "Unexpected dangling \"_\" in \"" + identifier + "\".");
         }
     }
 
@@ -54,7 +54,7 @@ module.exports = function(context) {
 
         if (typeof identifier !== "undefined" && hasTrailingUnderscore(identifier) &&
             !isSpecialCaseIdentifierForMemberExpression(identifier)) {
-            context.report(node, "Unexpected dangling '_' in '" + identifier + "'.");
+            context.report(node, "Unexpected dangling \"_\" in \"" + identifier + "\".");
         }
     }
 

--- a/tests/lib/rules/no-undef.js
+++ b/tests/lib/rules/no-undef.js
@@ -50,18 +50,18 @@ eslintTester.addRuleTest("lib/rules/no-undef", {
         { code: "import * as Warning from '../lib/warning'; var warn = new Warning('text');", ecmaFeatures: { modules: true } }
     ],
     invalid: [
-        { code: "a = 1;", errors: [{ message: "'a' is not defined.", type: "Identifier"}] },
-        { code: "var a = b;", errors: [{ message: "'b' is not defined.", type: "Identifier"}] },
-        { code: "function f() { b; }", errors: [{ message: "'b' is not defined.", type: "Identifier"}] },
-        { code: "/*global b:false*/ function f() { b = 1; }", errors: [{ message: "'b' is read only.", type: "Identifier"}] },
-        { code: "function f() { b = 1; }", global: { b: false }, errors: [{ message: "'b' is read only.", type: "Identifier"}] },
-        { code: "/*global b:false*/ function f() { b++; }", errors: [{ message: "'b' is read only.", type: "Identifier"}] },
-        { code: "/*global b*/ b = 1;", errors: [{ message: "'b' is read only.", type: "Identifier"}] },
-        { code: "/*global b:false*/ var b = 1;", errors: [{ message: "'b' is read only.", type: "Identifier"}] },
-        { code: "window;", errors: [{ message: "'window' is not defined.", type: "Identifier"}] },
-        { code: "require(\"a\");", errors: [{ message: "'require' is not defined.", type: "Identifier"}] },
-        { code: "Array = 1;", errors: [{ message: "'Array' is read only.", type: "Identifier"}] },
-        { code: "var React; React.render(<img attr={a} />);", args: [1, {vars: "all"}], errors: [{ message: "'a' is not defined." }], ecmaFeatures: { jsx: true } },
-        { code: "var React, App; React.render(<App attr={a} />);", args: [1, {vars: "all"}], errors: [{ message: "'a' is not defined." }], ecmaFeatures: { jsx: true } }
+        { code: "a = 1;", errors: [{ message: "\"a\" is not defined.", type: "Identifier"}] },
+        { code: "var a = b;", errors: [{ message: "\"b\" is not defined.", type: "Identifier"}] },
+        { code: "function f() { b; }", errors: [{ message: "\"b\" is not defined.", type: "Identifier"}] },
+        { code: "/*global b:false*/ function f() { b = 1; }", errors: [{ message: "\"b\" is read only.", type: "Identifier"}] },
+        { code: "function f() { b = 1; }", global: { b: false }, errors: [{ message: "\"b\" is read only.", type: "Identifier"}] },
+        { code: "/*global b:false*/ function f() { b++; }", errors: [{ message: "\"b\" is read only.", type: "Identifier"}] },
+        { code: "/*global b*/ b = 1;", errors: [{ message: "\"b\" is read only.", type: "Identifier"}] },
+        { code: "/*global b:false*/ var b = 1;", errors: [{ message: "\"b\" is read only.", type: "Identifier"}] },
+        { code: "window;", errors: [{ message: "\"window\" is not defined.", type: "Identifier"}] },
+        { code: "require(\"a\");", errors: [{ message: "\"require\" is not defined.", type: "Identifier"}] },
+        { code: "Array = 1;", errors: [{ message: "\"Array\" is read only.", type: "Identifier"}] },
+        { code: "var React; React.render(<img attr={a} />);", args: [1, {vars: "all"}], errors: [{ message: "\"a\" is not defined." }], ecmaFeatures: { jsx: true } },
+        { code: "var React, App; React.render(<App attr={a} />);", args: [1, {vars: "all"}], errors: [{ message: "\"a\" is not defined." }], ecmaFeatures: { jsx: true } }
     ]
 });

--- a/tests/lib/rules/no-underscore-dangle.js
+++ b/tests/lib/rules/no-underscore-dangle.js
@@ -28,11 +28,11 @@ eslintTester.addRuleTest("lib/rules/no-underscore-dangle", {
         { code: "export default function () {}", ecmaFeatures: { modules: true }}
     ],
     invalid: [
-        { code: "var _foo = 1", errors: [{ message: "Unexpected dangling '_' in '_foo'.", type: "VariableDeclarator"}] },
-        { code: "var foo_ = 1", errors: [{ message: "Unexpected dangling '_' in 'foo_'.", type: "VariableDeclarator"}] },
-        { code: "function _foo() {}", errors: [{ message: "Unexpected dangling '_' in '_foo'.", type: "FunctionDeclaration"}] },
-        { code: "function foo_() {}", errors: [{ message: "Unexpected dangling '_' in 'foo_'.", type: "FunctionDeclaration"}] },
-        { code: "var __proto__ = 1;", errors: [{ message: "Unexpected dangling '_' in '__proto__'.", type: "VariableDeclarator"}] },
-        { code: "foo._bar;", errors: [{ message: "Unexpected dangling '_' in '_bar'.", type: "MemberExpression"}] }
+        { code: "var _foo = 1", errors: [{ message: "Unexpected dangling \"_\" in \"_foo\".", type: "VariableDeclarator"}] },
+        { code: "var foo_ = 1", errors: [{ message: "Unexpected dangling \"_\" in \"foo_\".", type: "VariableDeclarator"}] },
+        { code: "function _foo() {}", errors: [{ message: "Unexpected dangling \"_\" in \"_foo\".", type: "FunctionDeclaration"}] },
+        { code: "function foo_() {}", errors: [{ message: "Unexpected dangling \"_\" in \"foo_\".", type: "FunctionDeclaration"}] },
+        { code: "var __proto__ = 1;", errors: [{ message: "Unexpected dangling \"_\" in \"__proto__\".", type: "VariableDeclarator"}] },
+        { code: "foo._bar;", errors: [{ message: "Unexpected dangling \"_\" in \"_bar\".", type: "MemberExpression"}] }
     ]
 });

